CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Development version
 
+GeoPandas no longer defaults to the PyGEOS engine and will use Shapely 2.0 by default, if
```

Comment:
```suggestion
GeoPandas will use Shapely 2.0 by default instead of PyGEOS when both Shapely >= 2.0 and PyGEOS are installed.  PyGEOS will continue to be used by default when PyGEOS is installed alongside Shapely < 2.0.  Support for PyGEOS and Shapely < 2.0 will be removed in GeoPandas 1.0.  
```